addon/serialize-query-params.js

```diff
@@ -0,0 +1,65 @@
+const RBRACKET = /\[\]$/;
```

Comment:
Can you create a `utils` directory and put this inside instead of it floating in `addon`?